### PR TITLE
[RI1 Ch07] User Manual initial content

### PR DESCRIPTION
**Pull request by mf4716**
Initial content and placeholders for contributor edits to Ch 7 for the Run-Book.
Solves: #408

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@mf4716 what is the status of this PR?

File: doc/ref_impl/cntt-ri/chapters/chapter07.md [hunk 1]
```diff
@@ -5,15 +5,64 @@
...
 
 <a name="7.3"></a>
-## 7.3 Heading
+## 7.3 Requirements Gathering
+
+Requirements gathering processes and steps:
+
+1.  RA Requirements Gathering process
+2.  Infra Requirements and Selection Process
+
```
[the remark this answers is not in the document]
Comment (mf4716) [reply doc/ref_impl/cntt-ri/chapters/chapter07.md]: That is fine.  The purpose is to document the requirements gathering process and steps, so RI Core Ch02, "RI Requirements" is a good location for this information.

File: doc/ref_impl/cntt-ri/chapters/chapter07.md [hunk 2]
```diff
@@ -5,15 +5,64 @@
...
+<a name="7.4"></a>
+## 7.4 Access & Connectivity
+
+Logical steps for lab access and connectivity.
+
+Sample steps provided mimic those utilized for POD10 test lab access.
```
[the remark this answers is not in the document]
Comment (mf4716) [reply doc/ref_impl/cntt-ri/chapters/chapter07.md]: That is fine.  The purpose is to document the steps for lab access and connectivity used for POD10, which can be repeated to a certain degree across any shared, community, lab.  Although, there may be subtle differences in some labs, the goal is to capture specific steps for one shared Intel POD, and provide means to receive assistance if steps differ for another lab.  RI Labs Ch06, "Lab Operations" is a good location for this information.

**Comment by mf4716** [comment]
Resolved open questions.  Asking @markshostak @rrajesh70 @fuqiao123 for reviews and approvals - please.

**Comment by mf4716** [comment]
@markshostak @rrajesh70 @fuqiao123 - please review approve/comment please.

File: doc/ref_impl/cntt-ri/chapters/chapter07.md [hunk 3]
```diff
@@ -5,15 +5,64 @@
...
+* [7.3 Requirements Gathering](#7.3)
+* [7.4 Access and Connectivity](#7.4)
+* [7.5 Descriptor File Preparations](#7.5)
+* [7.6 Deployment Installer & Install Steps](#7.6)
+* [7.7 Deployment Validations](#7.7)
+ 
 <a name="7.1"></a>
 ## 7.1 Introduction
 
+The purpose of this chapter is to establish an operational run-book containing sequences and steps explaining, wiht enough detail, how to install a Reference Implementation (RI).
```
Comment (deebhatia) [inline doc/ref_impl/cntt-ri/chapters/chapter07.md]: wiht -> with

File: doc/ref_impl/cntt-ri/chapters/chapter07.md [hunk 4]
```diff
@@ -5,15 +5,64 @@
...
+
+The term Run-Book is synonymous with "Cookbook", and either team will be used interchangeably to indicate that if a user follows the steps or procedures in the "book", the expected result will be an RI identical to the lab utilized by CNTT for the instation and certificaiton of RI-1.
```
Comment (deebhatia) [inline doc/ref_impl/cntt-ri/chapters/chapter07.md]: team -> term

**Comment by mf4716** [comment]
Can we get additional reviews/approvals please?  

@deebhatia , @fuqiao123 , @collivier , @markshostak

File: doc/ref_impl/cntt-ri/chapters/chapter07.md [hunk 5]
```diff
@@ -5,15 +5,84 @@
...
+* [7.8 CICD Tool Chain (use of, process, and scripts)](#7.8)
+* [7.9 Jenkins Setup & Job Creation](#7.9)
+* [7.10 Compliance Validation (steps, process)](#7.10)
+ 
 <a name="7.1"></a>
 ## 7.1 Introduction
 
+The purpose of this chapter is to establish an operational run-book containing sequences and steps explaining, with enough detail, how to install a Reference Implementation (RI).
+
+The term Run-Book is synonymous with "Cookbook", and either term will be used interchangeably to indicate that if a user follows the steps or procedures in the "book", the expected result will be an RI identical to the lab utilized by CNTT for the instation and certificaiton of RI-1.
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri/chapters/chapter07.md]: "... the expected result will be an RI identical to the lab utilized by CNTT for the instation and certificaiton of RI-1" -- will the run-book not be utilised in the installation of RI-1?

Typos: instantiation and certification.

**Comment by mf4716** [comment]
Thanks @markshostak - we'll refer to this as a (the) Cookbook (CB) as we iterate on the material.  Appreciate the feedback.

**Comment by mf4716** [comment]
@collivier @rabiabdel - can one of you review/approve please?  Says, "Code Owner review required - Merging is blocked".  Thanks in advance

**Pull request merged**